docs/development.md

```diff
@@ -0,0 +1,147 @@
+* [React](https://reactjs.org/)
+
+## Architecture
+
+The diagram below shows the overall data flow between the various moving components for Better Conversations.
+
+![Data Flows](http://www.plantuml.com/plantuml/proxy?cache=no&src=https://raw.githubusercontent.com/LBHackney-IT/coronavirus-frontdoor-snapshot/master/docs/Data-Flow.iuml)
+
+### AWS Accounts 
+
+This project is hosted in the Hackney AWS accounts, depending on the stage you're looking for.
+
+* Production: `ProductionAPIs`
+* Staging: `StagingAPIs`
+
+![AWS Architecture](http://www.plantuml.com/plantuml/proxy?cache=no&src=https://raw.githubusercontent.com/LBHackney-IT/coronavirus-frontdoor-snapshot/master/docs/infrastructure.iuml)
+
+
+## Integration with other tools
+
+There is currently an integration with the "I Need Help" tool.
+
+While the "I Need Help" tool has been replaced by "Here To Help", the tool remains accessible.
+
+It has an integration with Better Conversations as documented [here](https://docs.google.com/document/d/1jzpirzR3U8pDOwy_H-sI3jtZi9Or_2nPQH6nCW1lEuY/edit#heading=h.u4foorjj4xue)
+
+## Getting started
+
+1. Install the project dependencies
+   ```
+   yarn
+   ```
+   and
+   `mkdir dynamodblocal`
+   if the 'dynamodblocal' directory does not exist.
+2. Create a `.env` file based off of the `.env.sample` that exists.
+3. [Set up DynamoDB Local](https://docs.aws.amazon.com/amazondynamodb/latest/developerguide/DynamoDBLocal.html)
+   or start a local dynamodb container with Docker like so: `docker-compose up`
+4. Configure AWS cli with `aws configure`
+5. Create local DynamoDB plans table
+   ```bash
+   aws dynamodb create-table --cli-input-json file://./config/tables/referrals.json --endpoint-url http://localhost:8000
+   aws dynamodb create-table --cli-input-json file://./config/tables/conversations.json --endpoint-url http://localhost:8000 > /dev/null
+   ```
+6. Start running your local copy of the application.
+   ```
+   yarn dev
+   ```
+7. Open `http://localhost:3000` to verify the page opens, if that is the case your local setup is complete.
+
+## Working with DynamoDB locally
+
+If you need to view, edit or delete data from your local copy of DynamoDB when working on a feature
+there is a useful admin tool you can run.
+
+```(bash)
+yarn dynamo-admin
+```
+
+## Testing
+
+### Unit tests
+
+Unit tests are written with Jest, we are using `@testing-library/react` for React component testing.
+
+```(bash)
+yarn unit-test
+```
+
+### Integration tests
+
+Integration tests are written with Cypress.
+
+To run integration tests:
+
+1. Set these env variables to the following values:
+```(bash)
+AIRTABLE_API_KEY=xx
+AIRTABLE_BASE_ID=baseId123
+AIRTABLE_BASE_URL=http://localhost:8085
+AIRTABLE_PROMPTS_API_KEY=xx
+AIRTABLE_PROMPTS_BASE_ID=baseId456
+AIRTABLE_PROMPTS_TABLE_NAME="Conversational Prompts"
+AIRTABLE_TABLE_NAMES="Service directory"
+FSS_PUBLIC_API_URL=http://localhost:8085/fss
+HOTJAR_ID=12345
+INH_URL=https://inh-admin-test.hackney.gov.uk
+```
```

Comment:
Maybe we should just direct people to `.env.sample`? _(to reduce the amount of place we need to change list of env vars in)_.

Or if they have to be separate, then create `.env.integration`?

Reply:
Completely agree with the intention.

I think you're right in that it needs to be different because you'll probably want to be developing against the real services, rather than the fake ones - it also adds complexity around boot up.

Some .env plugins will end up loading all of the `.env.*` files, and I'm not sure about the ordering, so I'd be a little nervous about doing that and ending up breaking the live environment